src/commands/Testing/spawn.ts

```diff
@@ -1,20 +1,23 @@
 import { KlasaMessage, CommandStore } from 'klasa';
 
 import { BotCommand } from '../../lib/BotCommand';
-import getOSItem from '../../lib/util/getOSItem';
+import { Item } from 'oldschooljs/dist/meta/types';
+import { Bank } from '../../lib/types';
+import createReadableItemListFromBank from '../../lib/util/createReadableItemListFromTuple';
+import { Emoji } from '../../lib/constants';
 
 export default class extends BotCommand {
 	public constructor(store: CommandStore, file: string[], directory: string) {
 		super(store, file, directory, {
 			cooldown: 1,
-			usage: '[number:int{1,100}] <itemname:...string>',
+			usage: '[qty:integer{1,1000000}] (item:...item)',
 			usageDelim: ' ',
 			oneAtTime: true
 		});
 		this.enabled = !this.client.production;
 	}
 
-	async run(msg: KlasaMessage, [qtyOrName, itemNamePossible]: [number, number | undefined]) {
+	async run(msg: KlasaMessage, [qty = 1, item]: [number, Item[]]) {
```

Comment:
if `item` is always an array of items, it should probably be called `items` or `itemArray`